app/models/user.rb

```diff
@@ -931,9 +931,10 @@ def self.erase_user(id)
   # Does user have any unshown naming notifications?
   # (I'm thoroughly confused about what role the observation plays in this
   # complicated set of pages. -JPH)
+  # 2015-10-12 - JDC as currently written, observation appears to be lint
```

Comment:
1.  Will delete my comment.  
2.  I meant "lint" in the same way as used by Rubocop.  
Here, `has_unshown_naming_notifications?(observation=nil)` doesn't seem to do anything with its argument.
3. I'd prefer not to fix it as part of this PR, as fixing it seems unrelated, and I think I've already put too much stuff into this PR.  I will create a Pivotal Issue so we can deal with it in due course.  (Moreover, I cannot resolve the confusion right now because I'm also confused about the intent of the method.  Is it trying to determine if the user has (a) any unshown naming notifications, or (b) any unshown naming notifications for a single observation?